components/camel-pulsar/src/main/docs/pulsar-component.adoc

```diff
@@ -98,7 +98,7 @@ with the following path and query parameters:
 | *maxPendingMessagesAcross Partitions* (producer) | Set the number of max pending messages across all the partitions. Default is 50000. | 50000 | int
 | *messageRouter* (producer) | Set a custom Message Router. |  | MessageRouter
 | *messageRoutingMode* (producer) | Set the message routing mode for the producer. | RoundRobinPartition | MessageRoutingMode
-| *producerName* (producer) | Name of the producer | default-producer | String
+| *producerName* (producer) | Name of the producer, if unset lets Pulsar select a unique identifier | | String
```

Comment:
The doc changes in this table should happen in the source code, also it should be `an unique`

Reply:
Oh ok, I didn't think this was automated. Is it only the default value field that is done through the source code or is there some file I'm missing that contains the description? 

I disagree with `an unique` given the first syllable is pronounced like `you` which is not a vowel sound.

Reply:
Figured out how to do the docs. It's not exactly clear the maven command to run to generate the new one, but I think I've figured it out.